hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/TestFsDatasetImpl.java

```diff
@@ -1919,4 +1903,54 @@ public void delayDeleteReplica() {
       DataNodeFaultInjector.set(oldInjector);
     }
   }
+  @Test
```

Comment:
1. Leave one empty line between methods.
2. Please add some javadoc for this tests about what it will cover.